include/zephyr/net/ethernet.h

```diff
@@ -204,6 +204,12 @@ enum ethernet_hw_caps {
 
 	/** TX-Injection supported */
 	ETHERNET_TXINJECTION_MODE	= BIT(20),
+
+	/** 2.5 Gbits link supported */
+	ETHERNET_LINK_2_5GBASE_T	= BIT(21),
+
+	/** 5 Gbits link supported */
+	ETHERNET_LINK_5GBASE_T		= BIT(22),
```

Comment:
```suggestion
	ETHERNET_LINK_2500BASE_T	= BIT(21),

	/** 5 Gbits link supported */
	ETHERNET_LINK_5000BASE_T	= BIT(22),
```

Seems to be more in line with other types.